typescript/infra/config/environments/mainnet3/gasPrices.json

```diff
@@ -1,17 +1,17 @@
 {
-  "arbitrum": "0.1",
+  "arbitrum": "1",
   "avalanche": "43.212830197",
-  "bsc": "1.350070319",
-  "celo": "10.0",
-  "ethereum": "26.346912847",
-  "mantapacific": "0.100000057",
-  "moonbeam": "125.0",
-  "optimism": "0.003225814",
-  "polygon": "61.601287856",
-  "gnosis": "1.852997796",
-  "base": "0.0010003",
-  "scroll": "0.46",
-  "polygonzkevm": "3.95",
+  "bsc": "7",
```

Comment:
did you update just gasPrices or also token exchange rates?

also are these values handcrafted or were they generated by the tooling?

Do you see yourself merging this into main or that https://github.com/hyperlane-xyz/issues/issues/1178 will take this over / supersede this